src/tests/unittests/measurements/processing/test_process_sawtooth_2d.py

```diff
@@ -62,8 +63,8 @@ def __dummy_scope_data(time_data, sawteeth_count, period, width):
 
     @staticmethod
     def __dummy_data_array(set_array: DataArray, scope_data: np.ndarray, channel_index: int = 1, trace_number: int = 1):
-        idenifier = 'ScopeTrace_{:03d}'.format(trace_number)
-        label = 'Channel_{}'.format(channel_index)
+        idenifier = f'ScopeTrace_{trace_number:03d}'
```

Comment:
Idenifier->Identifier

Reply:
The spelling mistake was already in the original code and this is part of the deprecated class.